app/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,212 @@
+    <string name="setting_card_icons">اختر رموز الأهداف</string>
+    <string name="setting_card_images">اختر صور الأهداف</string>
+    <string name="setting_theme_mode">وضع التطبيق</string>
+    <string name="setting_theme_light">نهاري</string>
+    <string name="setting_theme_dark">ليلي</string>
+    <string name="setting_theme_system">افتراضي النظام</string>
+    <string name="setting_theme_onboarding_title">اختر الوضع المفضل لديك.</string>
+    <string name="setting_theme_onboarding_desc">افتح الإعدادات واختر بين الوضع النهاري أو الليلي.</string>
+    <string name="setting_goal_card_style">نمط بطاقات الأهداف</string>
+    <string name="setting_goal_card_style_compact">مضغوط</string>
+    <string name="setting_goal_card_style_extended">ممتد</string>
+    <string name="setting_goal_card_onboarding_title">اختر نمط البطاقة المفضل لديك.</string>
+    <string name="setting_goal_card_onboarding_desc">بطاقات الأهداف تأتي بأنماط متعددة. جربها جميعًا واختار ما يناسبك.</string>
+    <string name="setting_currency">اختر العملة المفضلة لديك</string>
+    <string name="setting_currency_onboarding_title">العملة التي تفضلها؟</string>
+    <string name="setting_currency_onboarding_desc">GreenStash يدعم عدة عملات. اختر العملة التي تفضلها في الإعدادات.</string>
+    <string name="setting_enable_biometrics">تفعيل قفل التطبيق باستخدام القياسات الحيوية؟</string>
+    <string name="setting_enable_biometrics_desc">أدخل الأمان الإضافي باستخدام القفل البيومتري.</string>
+    <string name="setting_enable_notifications">تفعيل الإشعارات؟</string>
+    <string name="setting_enable_notifications_desc">استمر في التحديثات حول تقدم هدفك باستخدام الإشعارات.</string>
+    <string name="setting_backups">النسخ الاحتياطي</string>
+    <string name="setting_backups_desc">حافظ على أمان بياناتك من خلال النسخ الاحتياطي المنتظم.</string>
+    <string name="setting_about">حول</string>
+    <string name="setting_about_desc">اعرف المزيد حول تطبيق GreenStash.</string>
+    <string name="setting_privacy">الخصوصية</string>
+    <string name="setting_privacy_desc">قراءة سياسة الخصوصية.</string>
+    <string name="setting_version">إصدار التطبيق</string>
+
+    <!-- Backup Screen -->
+    <string name="backup_title">النسخ الاحتياطي</string>
+    <string name="backup_import">استيراد النسخ الاحتياطي</string>
+    <string name="backup_export">تصدير النسخ الاحتياطي</string>
+    <string name="backup_export_desc">احتفظ بنسخة من بياناتك.</string>
+    <string name="backup_import_desc">استيراد البيانات المحفوظة سابقًا.</string>
+    <string name="backup_export_success">تم التصدير بنجاح.</string>
+    <string name="backup_import_success">تم الاستيراد بنجاح.</string>
+    <string name="backup_export_failed">فشل التصدير.</string>
+    <string name="backup_import_failed">فشل الاستيراد.</string>
+
+    <!-- Archive Screen -->
+    <string name="archive_screen_header">الأرشيف</string>
+    <string name="archive_empty_placeholder">لا توجد أهداف مؤرشفة حتى الآن.</string>
+
+    <!-- Rate Us Dialog -->
```

Comment:
I don't see anything like this in the base English `strings.xml`. It seems the translations are not complete and are not in the same order as the base English strings. This is the reason why the build was failing. Please do a complete translation when adding a new language; otherwise, this results in an inconsistent user experience. Also, maintain the order of sections of strings the same as in the base `strings.xml` file so it's easier to find, update, and add new strings in the future.

Please take a look at the base strings file [here](https://github.com/Pool-Of-Tears/GreenStash/blob/main/app/src/main/res/values/strings.xml) and translate accordingly.

Reply:
Excuse my ignorance! I'll do when I'll have some free time!

Reply:
Sure, there's no hurry :)